src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -362,6 +363,40 @@ static void uct_ib_mlx5_devx_mr_lru_cleanup(uct_ib_mlx5_md_t *md)
     kh_destroy_inplace(rkeys, &md->lru_rkeys.hash);
 }
 
+static ucs_status_t
+uct_ib_mlx5_devx_reg_ksm_data(uct_ib_mlx5_md_t *md, uct_ib_mlx5_devx_mem_t *memh,
```

Comment:
minor: why do we need `data` in the naming? maybe:
`uct_ib_mlx5_devx_reg_ksm`
`uct_ib_mlx5_devx_reg_ksm_mt`
`uct_ib_mlx5_devx_reg_ksm_contig`